scripts/index.js

```diff
@@ -40,6 +53,24 @@ const pageContent = async (url, options) => {
   return content
 }
 
+
+// const writeToCSV = (statuses) => {
```

Comment:
Same here

Reply:
We are not calling this function any more, might be in the future. That's why I uncommented it.

Reply:
cool, now I'm clear